api/outbox.py

```diff
@@ -0,0 +1,43 @@
+from ..request import Request, Response
+from ..interface.defs import Endpoint, Method
+from .generic import GenericResponse, GenericTransaction
+
+
+class NotificationEntry:
+    def __init__(self, uid: int, type: int, change: int, record_id: int):
+        self.uid = uid
+        self.type = type
+        self.change = change
```

Comment:
It's ok now :) 
I wont force your hand, it's nice to know though: the best would be to specify the `type` and `change` enum and use typed python with it. Then it would be the best :) You would be able to check if value is of enum, not if  it's `1` (like it's done in the test)

Saying that - I don't want to request changes after changes, especially that it looks good enough.

Reply:
Enum classes for  `type` and change `added`